storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -315,13 +315,6 @@
     @IsInteger
     @IsPositiveNumber
     public static final String TOPOLOGY_RAS_CONSTRAINT_MAX_STATE_SEARCH = "topology.ras.constraint.max.state.search";
-    /**
-     * The maximum number of states that will be searched looking for a solution in the constraint solver strategy.
-     * Backward compatibility config value for old topologies
-     */
-    @IsInteger
-    @IsPositiveNumber
-    public static final String TOPOLOGY_RAS_CONSTRAINT_MAX_STATE_TRAVERSAL = "topology.ras.constraint.max.state.traversal";
```

Comment:
This was introduced by mistake https://github.com/apache/storm/pull/3056. We have this config on 0.10 storm in internal mirrors.  But this is not an issue on community Storm codebase